### PR TITLE
#155738450 Setup Circleci, Codecov and Code climate

### PR DESCRIPTION
**Pull request by fahadmak**
#### What does this PR do?
- Sets up circle ci, code cov and code climate
#### Description of Task to be completed?
- Create config.yml file 
- Link repository to circle ci
- Link repository to code climate
- Link repository to codecov
#### What are the relevant pivotal tracker stories?
[#155738450](https://www.pivotaltracker.com/story/show/155738450)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,4 +1,18 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.dicedmelon.gradle:jacoco-android:0.1.4'
+    }
+}
+
```
Comment (johngorithm) [inline app/build.gradle]: The above script should be in the project level `build.gradle` not in the app level. Kindly, fix that ..

**Review by johngorithm — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (fahadmak) [reply app/build.gradle]: thanks for that feedback

**Comment by johngorithm** [comment]
LGTM

**Comment by leonardnjura** [comment]
LGTM.

**Pull request merged**